docs/config-ownership.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[config-file-ownership]]
+==== Configuration file ownership
+
+On systems with POSIX file permissions,
+the {beatname_uc} configuration file is subject to ownership and file permission checks.
+These checks prevent unauthorized users from providing or modifying configurations that are run by {beatname_uc}.
+
+When installed via an RPM or DEB package,
+the configuration file at +/etc/{beatname_lc}/{beatname_lc}.yml+ will be owned by +{beatname_lc}+,
+and have file permissions of `0600` (`-rw-------`).
+
+{beatname_uc} will only start if the configuration file is owned by the user running the process,
+or by running as root with configuration ownership set to `root:root`
+
+You may encounter the following errors if your configuration file fails these checks:
+
+["source", "systemd", subs="attributes"]
+-----
+Exiting: error loading config file: config file ("/etc/{beatname_lc}/{beatname_lc}.yml")
+must be owned by the user identifier (uid=1000) or root
+-----
+
+To correct this problem you can change the ownership of the configuration file with:
++chown {beatname_lc}:{beatname_lc} /etc/{beatname_lc}/{beatname_lc}.yml+.
+
+You can also make the config ownership world readable, although this is not recommended:
++chown root:root /etc/{beatname_lc}/{beatname_lc}.yml`+.
```

Comment:
I don't really follow this - the command makes root the owner, it doesn't make it world readable.

Reply:
Oh, the following bits are related to this too? It read to me like the `chown root:root` was meant to make it world readable.

Reply:
Good call. That is confusing.